### PR TITLE
refactor: replace FontAwesome with Icon everywhere

### PR DESCRIPTION
**Pull request by rast-mk1**

### CODE REVIEW

File: app/components/Icon/index.js [hunk 1]
```diff
@@ -12,6 +12,7 @@ import {
   faChevronRight,
   faChevronDown,
   faGear,
+  faSortDown,
 } from '@fortawesome/free-solid-svg-icons';
 import { faFacebookF, faInstagram, faTwitter, faYoutube } from '@fortawesome/free-brands-svg-icons';
 const iconMap = {
```
Comment (weronika299) [inline app/components/Icon/index.js:18]: Why do you have declared Icons sometimes as string and sometimes not?

Comment (rast-mk1) [reply app/components/Icon/index.js:18]: FontAwesome icons are imported in this file and then "translated" to string inside `iconMap`. So `<FontAwesome icon={faTwitter} />` is basically the same as `<Icon type="twitter" />` and importing `faTwitter` takes place only in `Icon/index.js`.

Comment (rast-mk1) [reply app/components/Icon/index.js:18]: Ah, yes: Prettier changes single words like that and leaves multiple-word names inside quotes :)

**Review by hiimsagan — APPROVED** [review]
Looks good

File: app/components/Icon/index.js (hunk 1 above)
Comment (weronika299) [reply app/components/Icon/index.js:18]: Aaa ok :) But is it work?  I ask because in mockLists file you have it inside quotes. :)

Comment (rast-mk1) [reply app/components/Icon/index.js:18]: Yes, it works fine.

Comment (weronika299) [reply app/components/Icon/index.js:18]: Ok, great :D

**Review by weronika299 — APPROVED** [review]

**Pull request merged**